### ISSUE TITLE
Missing Kotlin/StringsKit

### ISSUE THREAD

**Issue by ebergerson** [abrookins/WrapToColumn#20]
Just installed the plugin to the latest version of pycharm using install from disk on the WrapToColumn.jar.  I get this error when I try to wrap to column:

```
8/2/16
10:55 AM	NoClassDefFoundError: kotlin/StringsKt
```

I see the kotlin-runtime.jar in the WrapToColumn distribution, but there was no indication of what to do with it.  Does it have to be installed somewhere?

Thanks.

**Comment by ebergerson** [comment]
BTW, I just looked in the PyCharm lib directory and a kotlin runtime jar is already there.

```
ingram:/Applications/PyCharm.app/Contents/lib
eb$ ls k*
kotlin-reflect.jar	kotlin-runtime.jar
ingram:/Applications/PyCharm.app/Contents/lib
```

**Comment by ebergerson** [comment]
Sorry, mistake, didn't mean to close the issue.  So, it still posts the exception above when I go to run wrap to column, despite the kotlin-runtime.jar being in the PyCharm lib directory.

**Comment by abrookins** [comment]
Very interesting! I'll check this out soon. What operating system are you using? And is that PyCharm 2016.2 (you said latest, so I assume it is)? Have you tried installing the plugin via the JetBrains plugin repository? I.e., Preferences (Settings) -> Plugins -> Browse Repositories -> search for 'wrap'.

**Comment by ebergerson** [comment]
That did it.

Yes, PyCharm 2016.2.  Mac OS 10.10.5 Yosemite. 

I installed it from the download because I did not see it in the plugins list.  I realize now that I didn't look in "Browse Repositories".  I uninstalled, restarted, installed as you instructed, restarted again. 

Now it works.

Thanks so much.  Not only does it work, but it does the right thing when you are in a mark up list!

For example:

```
- this is a short line
- The cursor is in this line and it is very long
- this is a short line, no blank lines between any of this
```
When I wrap to column the middle line wraps without including the line before or after.

Very impressive.  Nice job.

Now if only I could highlight all of the lines and they all individually wrap, but not into each other!

Thank your for contributing this to the community.
